client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -304,9 +319,24 @@ export class CommentNavigation extends Component {
 					) }
 
 					{ hasComments && (
-						<Button compact onClick={ toggleBulkMode }>
-							{ translate( 'Bulk edit' ) }
-						</Button>
+						<Fragment>
```

Comment:
We could use the short syntax:
```suggestion
						<>
```